ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent { label "docker-gpu"}
+    environment {
+        DOCKER_BUILDKIT=1
+        UBUNTU_BASE_TAG=20.04
+    }
+    stages {
+        stage("Capture Build Info"){
+            steps {
+                rtBuildInfo (captureEnv: true)
+            }
+        }
+        stage("Build Docker Image") {
+            steps {
+                sh script: """
+                    docker build \
+                        --progress=plain \
+                        --tag pynvml:${BUILD_NUMBER} \
+                        --file ci/Dockerfile .
+                    """
+            }
+        }
+        stage("Build Wheels & Test"){
+            steps {
+                sh script: """
+                    docker run \
+                        --interactive \
+                        --name pynvml \
+                        --gpus all pynvml:${BUILD_NUMBER}
+                    """
+                sh script: "docker cp pynvml:/sandbox/artifacts ."
```

Comment:
Using `docker cp` to avoid permission issues when using mount. `--output` is not available outside of `build`.